src/models/chan.js

```diff
@@ -25,7 +25,7 @@ function Chan(attr) {
 		type: Chan.Type.CHANNEL,
 		firstUnread: 0,
 		unread: 0,
-		highlight: false,
+		highlight: 0,
```

Comment:
Kind of a nitpick, but could you rename this into `highlights` or `highlightCount` then?

Reply:
This requires finding all the instances (client included), I'd rather not.

Reply:
Apart from those you have modified in this PR, the only other occurrence of this that I found is https://github.com/thelounge/lounge/blob/master/client/views/chan.tpl#L3

Reply:
There is definitely more.

Reply:
All others that I found are for `Msg.highlight`.